libopae/plugins/xfpga/plugin.c

```diff
@@ -73,6 +73,9 @@ int __FPGA_API__ opae_plugin_configure(opae_api_adapter_table *adapter,
 {
 	UNUSED_PARAM(jsonConfig);
 
+	if (adapter == NULL)
```

Comment:
This check isn't needed.  See pluginmgr.c:679
```c
if (!adapter) {
   OPAE_ERR("malloc failed");
   return ++errors;
}
```

Reply:
EXPECT_EQ(opae_plugin_configure(adapter_table, NULL), 0); -> PASS

test case EXPECT_EQ(opae_plugin_configure(NULL, NULL), -1); -> Core dump.
No Null check

Reply:
The problem is that you've inserted a case that does not exist in the code, and then you've added test cases around it.  I'm just pointing out that neither the check nor the test of the check is needed, based on the calling code.  Sure, theoretically you could call this function with a NULL pointer, but it isn't happening today.  Let's not create it.

Reply:
Removed test cases and plugin check .